armory/utils/external_repo.py

```diff
@@ -24,7 +42,6 @@ def download_and_extract_repo(
     repositories project root will be added to the sys path.
 
     Private repositories require an `ARMORY_GITHUB_TOKEN` environment variable.
-    :param external_repo_name: String name of "organization/repo-name"
```

Comment:
Reason why this was deleted?

Reply:
Thought it was misleading since it should say:
```
String name of "organization/repo-name" or "organization/repo-name@branch"
```

Then I realized we hadn't documented the other params so wasn't sure it was worth updating since that information is available in the documentation. I can update it if you'd prefer

Reply:
Let's just update it.

Reply:
Done. I need to update https://github.com/twosixlabs/armory/blob/master/docs/external_repos.md as well though